### ISSUE TITLE
Issues with mal api

### ISSUE THREAD

**Issue by EloMonarch** [darenliang/mal-api#9]
Hello,
So I wanted to start a project with tkinter and mal api to do some stuff not sure what to do yet but I installed mal-api using "pip install mal-api" and then imported it using both "import mal" and "from mal import *". But I dont seem to find any of the available properties? I try searching for anime and all even used the code from the example "anime = Anime(1)" and "print(anime.score)" but I dont have anything named Anime or manga or anything tbh, I only have AnimeLabel when I try using the anime property.
Thanks in advance!
 [I am prety new in coding (saying that to prevent flaming if what I say is idiotic or something I didnt pay attetion to)]

Edit: Should've prob named it "Issues using mal api" cause the title might be a bit misleading
![Στιγμιότυπο οθόνης (94)](https://user-images.githubusercontent.com/30046785/101839025-52573e00-3b4a-11eb-8d8e-109a76f680a3.png)

**Comment by darenliang** [comment]
Hi @EloMonarch,

Can you confirm that `Anime` (make sure it is capitalized) is detected by auto-complete?
Any anime object created should have its field properties displayed through auto-complete.

**Comment by EloMonarch** [comment]
@darenliang 
Ah sorry but, I got confused with your answer 
How can I check if it's detected by auto-complete? By what I understood you mean if it pops up when I type mal. ? Because if that's what you meant, no it doesnt, nor can I use any property from the documentation.

What I get is:
anime = Anime(1)
NameError: name 'Anime' is not defined

**Comment by darenliang** [comment]
Could you run the following command in your terminal?

`pip freeze`

It should return a list of packages with their versions. I wanna double make sure that the correct package is installed.

**Comment by EloMonarch** [comment]
Here you go:
![Στιγμιότυπο οθόνης (95)](https://user-images.githubusercontent.com/30046785/101840491-df02fb80-3b4c-11eb-8c43-535994d44f0a.png)

**Comment by darenliang** [comment]
It looks like the package is installed, however, I'm wondering if you are using a different python environment.

Could you use the command `where python` in the command-line?

**Comment by EloMonarch** [comment]
I get this:

**Comment by darenliang** [comment]
One other thing, could you click on the python environment button at the bottom left of vscode?

Just wanna make sure that the python environment you are using in vscode is the same environment.

![image](https://user-images.githubusercontent.com/44593263/101841622-4cdf0100-3b14-11eb-9d3c-64379804c3b3.png)

**Comment by EloMonarch** [comment]
![Στιγμιότυπο οθόνης (97)](https://user-images.githubusercontent.com/30046785/101841775-4ae66380-3b4f-11eb-84e9-b77d0927fe9f.png)

**Comment by darenliang** [comment]
It looks like you have more than one python interpreter installed.

Could you try `python --version` in your terminal?

**Comment by EloMonarch** [comment]
The output is python 3.8.6

**Comment by darenliang** [comment]
I'm not sure what is the problem that is occurring, I've tried it locally and it seems to work.

My best guess is that your python environment is not detecting `mal-api`.

Normally, I would use a virtual environment when I install python packages and maybe it might work in this case.

Inside your work folder, run the following commands:
```
python -m venv venv
venv\Scripts\activate.bat

pip install mal-api
```

Create a python file (`main.py`) with the following:
```py
from mal import Anime

anime = Anime(1)

print(anime.score)
```

Run the python file:
```
python main.py
```

This might not be the best place to do troubleshooting so if it is preferable we could look over this issue somewhere else (e.g. Discord, etc.).

**Comment by EloMonarch** [comment]
Hmmm, it doesn't even find mal..
![Στιγμιότυπο οθόνης (98)](https://user-images.githubusercontent.com/30046785/101843664-62275000-3b53-11eb-915d-1e30cbbedac4.png)

Sure thing! Could you give me your discord?

**Comment by EloMonarch** [comment]
wait what, now I can't find more modules, like projects I did today do not work..

**Comment by EloMonarch** [comment]
I uninstalled it and installed it then rebooted my computer and I managed to make it work, but now when I do anime = Anime(1) and print(anime.title) I get this: <property object at 0x0000014F3146F400>

**Comment by darenliang** [comment]
Are you sure you didn't capitalize `anime`?

```
# incorrect
print(Anime.title)

# correct
print(anime.title)
```

**Comment by EloMonarch** [comment]
@darenliang 
Solved: When I installed it I named the file "mal" which game me the option to make import and mal would show up as the file name NOT the libray thus nothing was being imported and when I had the property of "AnimeLabel" it was actually another var in the project. More over it might've runned the command with other python ver and fixed it after the reboot, now it works fine. Sorry for misleading info and wasting your time, all I can say is that you were awesome and thank you for your help and fast replies.
